builtin/providers/google/resource_container_cluster_test.go

```diff
@@ -339,6 +340,7 @@ resource "google_container_cluster" "with_node_config" {
 	node_config {
 		machine_type = "g1-small"
```

Comment:
The machine type here will need to be `n1-standard-1` (tests fail with "local SSDs require machine type n1-standard-1 or larger")